examples/cactus-example-cbdc-bridging-backend/CHANGELOG.md

```diff
@@ -1,27 +0,0 @@
-# Change Log
```

Comment:
please inlude the deletion of this example into a separate pr since we may need to get this code back